src/components/link/common/link.scss

```diff
@@ -1,8 +1,7 @@
 @use '../../core/styles' as sbb;
 
-// Default component properties, defined for :host. Properties which can not
-// travel the shadow boundary are defined through this mixin
-@include sbb.host-component-properties;
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
```

Comment:
I see no specific `display` rule on host; should it be added?

Reply:
It's because link.scss is always used together with either block-link-scss or inline-link.scss. they both have their own display property. inline-link: inline-block, and blocklink: block

Reply:
Oh, right :)